presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -168,7 +168,8 @@ public int run(List<QueryPair> queries)
                 try {
                     ((Closeable) eventClient).close();
                 }
-                catch (IOException ignored) { }
+                catch (IOException ignored) {
```

Comment:
why not:
```
catch (IOException) {}
```
?

Reply:
Because such is the prevailing style for empty catch bodies (every other place has new line).